### ISSUE TITLE
Front Page Sports Baseball '94 (1994)

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoDOS#3121]
This game has some interesting copy protection. Its triggered by the file dates changing but doesnt show itself until the 9th innings.

Final comment from @SmilingSpectre:

agathosdaimonso: if i  understand right now - i install it with Daum and then apply the patches with Daum also and that should work?
Spectre: Yes, agathos. At least, it worked for krc with Staging. :)

I wonder if any other front page sports games might be affected?

**Comment by exoscoriae** [comment]
thoughts and notes:
the 1.03 patches does not change the reported game version in the game. However, I was able to start the game without a cd mounted, and that is supposed to be one of the benefits of 1.03

Second, general midi sounds no different to me than sound blaster. I tried GUS and it sounded identical as well.

That is when I noticed that changed the sound driver settings in setsound only appears to change the sound effects card, not the music.

Might need to remove the sc-55 option, as I don't believe it actually works?

**Comment by Python-Exoproject** [comment]
I agree, the sound choice seems irrelevant so unless something is not setup correctly it can be removed I think. Dont forget the playlist.

**Comment by exoscoriae** [comment]
removed sound choice and removed from playlist

**Comment by Python-Exoproject** [comment]
Game has:

cls
@PLAYBALLPLAYBALL
exit

in the conf when it should only have:

cls
@PLAYBALL
exit

**Comment by exoscoriae** [comment]
weird that is still worked like that....
